android/KMEA/app/src/main/java/com/keyman/engine/KMKeyboard.java

```diff
@@ -167,9 +168,29 @@ protected boolean updateSelectionRange(int selStart, int selEnd) {
     InputConnection ic = KMManager.getInputConnection(this.keyboardType);
     if (ic != null) {
       ExtractedText icText = ic.getExtractedText(new ExtractedTextRequest(), 0);
+      String rawText = icText.text.toString();
       if (icText != null) {
-        updateText(icText.text.toString());
+        updateText(rawText.toString());
       }
```

Comment:
Prob should assign rawText if icText != null
```suggestion
      String rawText = "";
      if (icText != null) {
        rawText = icText.text.toString();
        updateText(rawText.toString());
      }
```